routes/web.php

```diff
@@ -14,5 +14,18 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('home');
+});
+
+Route::get('/template', function () {
+    return view('layout/app');
+});
+
+Route::get('/theWideMouthFrog', function () {
+    return view('challengers/theWideMouthFrog');
```

Comment:
Same here

Reply:
Line 24 has been changed, But line 25 is based on the file name.